### PR TITLE
No Favorites To Show , then Display Message

### LINKED ISSUE #643: If there are no favourites, show a message rather than showing a blank page.
**Describe the bug**

If there are no favourites added by a user, show a message rather than showing a blank page or either open the favourites page and there show a message saying there are no favourites selected by the user yet and give an option to view more projects by that author.

**To Reproduce**
1. Go to 'https://circuitverse.org/users/13353'
2. Click on 'View Favourites'
3. See error

**Expected behaviour**

An empty page doesn't look good. In the first place, it gives an impression of what is wrong with the page. So, better write that there re no favourites yet.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 78.0.3904.108

**Are you working on this issue? (Yes/No)**
No. For GCI 2019.

### PR DESCRIPTION
**Pull request by Shivansh2407**
Fixes #643 

#### Describe the changes you have made in this pr -
I have added a Message , so whenever there are no favorite/starred circuit by any user then it will display a message .
### Screenshots of the changes (If any) - 
Annotation 2020-02-06 122527

### CODE REVIEW

**Comment by Shivansh2407** [comment]
Will Squash the Commits Soon Once approved . Please Review the changes @satu0king

File: app/views/users/logix/favourites.html.erb [hunk 1]
```diff
@@ -8,8 +8,13 @@
   <div class ="row">
     <h2 class="card-title">Favourite Projects</h2>
   </div>
-
-  <div class="row ">
+  <% ratedproject = @user.rated_projects %>
+    <% if ratedproject.count == 0 %>
```
Comment (tachyons) [inline app/views/users/logix/favourites.html.erb]: no need to store this in a temp variable, just use if @user.projects.blank?

Comment (Shivansh2407) [reply app/views/users/logix/favourites.html.erb]: But @tachyons here it is asking for rated_projects (Favourites) so shouldn't be @user.rated_projects.blank ?

**Comment by Shivansh2407** [comment]
@tachyons please review and merge. 👍

**Comment by Ananthavijay** [comment]
@Shivansh2407 Do the changes requested by @tachyons

**Comment by Ananthavijay** [comment]
Center the text, because the page seems to be empty

**Comment by Shivansh2407** [comment]
Done 👍 
Capture

**Comment by Shivansh2407** [comment]
@tachyons Please review and Merge.

**Comment by tachyons** [comment]
Styles in CSS please

**Comment by Shivansh2407** [comment]
@tachyons I have Squashed and Commited the requested Changes. Please Review it.

**Comment by Shivansh2407** [comment]
@tachyons Please review and merge this PR 👍

File: app/views/users/logix/favourites.html.erb (hunk 1 above)
Comment (tachyons) [reply app/views/users/logix/favourites.html.erb]: Yes, I was just suggesting the syntax

**Comment by Shivansh2407** [comment]
Thank you @tachyons 🎉 👍

**Pull request merged**